src/components/chips/chips.spec.js

```diff
@@ -409,6 +409,71 @@ describe('<md-chips>', function() {
         }));
       });
 
+      describe('md-max-chips', function() {
```

Comment:
Now Passing (just said in the comment before :smile:, so please don't be to harsh). Updated the `check` to ->

```
MdChipsCtrl.prototype.hasMaxChips = function() {
  if (angular.isString(this.maxChips)) this.maxChips = parseInt(this.maxChips, 10) || 0;

  return this.maxChips > 0 && this.items.length >= this.maxChips;
};
```

If undefined the it will return false, because `undefined > 0` will be false.